backend/communications/templates/chat/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8"/>
+    <title>Chat Rooms</title>
+</head>
+<body>
+    What chat room would you like to enter?<br>
+    <input id="room-name-input" type="text" size="100"><br>
+    <input id="room-name-submit" type="button" value="Enter">
+
+    <script>
+        document.querySelector('#room-name-input').focus();
```

Comment:
You query the same input element 3 times. This is not efficient.
Firstly you have to query elements and later work with them.

`const roomNameInput = document.querySelector('#room-name-input')`

Remember that querySelector returns `null` or `Node`.
There is need for check before making anything with node.

```
if (!roomNameInput) {
      console.log("input node #room-name-input not found.)
      return;
}
```

later in code -> `roomNameInput.focus()`

This is how it should looks like:

```suggestion
      <script>
        const roomNameInput = document.querySelector('#room-name-input');
        const roomSubmitButton = document.querySelector('#room-name-submit');
        if (!roomNameInput || !roomSubmitButton) return;
        
        roomNameInput.onkeyup = function(e) {
            if (e.key === 'Enter') {  // enter, return
                roomSubmitButton.click();
            }
        };

        roomNameInput.onclick = function(e) {
            const roomName = roomNameInput.value;
            window.location.pathname = '/chat/' + roomName + '/';
        };
    </script>
```

Reply:
and also extend this version with suggestions from @VovaD199